django/core/management/utils.py

```diff
@@ -75,3 +76,37 @@ def find_command(cmd, path=None, pathext=None):
             if os.path.isfile(fext):
                 return fext
     return None
+
+
+def parse_model_labels(labels, source=None):
```

Comment:
I agree with your point that raising `CommandError` inside the command itself would be more clean, but it's a bit tricky, since we should maintain the proper exception message, which differs for different cases. So if it doesn't introduce a total mess, I'd keep the current version. As for making the output incompatible - since I don't know for sure noone depends on it, I'd prefer to keep the format the same.